src/styles/forms.scss

```diff
@@ -77,6 +77,33 @@ $input-text-color: #495057;
     margin: 0 10px 10px 0;
 }
 
+// Danger Zone -----------------------------------
+ 
+#danger-zone-container {
+    margin: 50px 10px 20px;
+    border-radius: 5px;
+    border: 1px solid red;
+    padding: 10px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+   
+    #danger-text {
+        color: black;
+        font-size: 24px;
+        word-break: break-word;
+        word-wrap: break-word;
+    }
+ 
+    .delete-mutation-test-suite-button, 
+    .delete-ag-test-suite-button,
+    .delete-ag-test-command-button {
```

Comment:
Instead of 3 different class names, let's just use one, like `.delete-button`.

Reply:
I did that originally, although that broke the build because of other code that depended on those specific classnames shown above. Would you want me to go edit the other files?

Broken Build from using a single variable name: https://travis-ci.org/eecs-autograder/ag-website-vue/builds/619939903?utm_source=github_status&utm_medium=notification
Original single variable classname: https://github.com/eecs-autograder/ag-website-vue/pull/346/commits/ca60873598c71ab13039151ae43fecc6688bf712

Reply:
Maybe you could add the `delete-button` class to the html and keep the classes that the tests depend on?

Reply:
Okay, that makes sense, thank you!